go/service/main.go

```diff
@@ -138,6 +138,11 @@ func (d *Service) Run() (err error) {
 		}
 	}
 
+	// Explicitly set fork type here based on KEYBASE_LABEL
+	if len(d.G().Env.GetLabel()) > 0 {
```

Comment:
Good point, thanks! It would be so much cleaner to check on the command line, but @gabriel has asked us not to.